config/routes.rb

```diff
@@ -6,12 +6,18 @@
   resources :availabilities, only: [:index] do
     collection do
       post '/', to: 'availabilities#update_availabilities'
+      get '/show_everyone', to: 'availabilities#show_everyone'
     end
   end
 
+  namespace :availabilities do
+    resources :places, only: %i[index edit update]
```

Comment:
I think you can put this resources on the `resources :availabilities` above

Reply:
But this is not for each availability, it's for overall availabilities, filtered by places. If I put this inside `resources :availabilities` then the URL will be `/availabilities/:id/places/:id` but I want `/availabilities/places/:id`

Reply:
Then put it inside the `collection` block

Reply:
But then that would be namespace clash with the places_controller.rb

Reply:
@donjar bump

Reply:
I don't get what you're trying to say - what's with the controller file?

![screenshot from 2018-04-19 16-48-08](https://user-images.githubusercontent.com/14049598/38981233-7aa21a86-43f1-11e8-96fa-da4846948e5e.png)

Isn't this what you intended?

Reply:
Yes, but it I put that under collection block under resources :availabilities, the controller is places not availabilities/places

Reply:
Oh ok. Hmm an alternative would be to change the controller actions to be `set_default_users` etc instead of the default `index` `edit` `update` etc. But yeah both are ok for me

Reply:
So would the current code be alright?

Reply:
Done